src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -120,7 +120,11 @@ public function startTestSuite($suite)
             $this->state = 0;
 
             if (!class_exists('Doctrine\Common\Annotations\AnnotationRegistry', false) && class_exists('Doctrine\Common\Annotations\AnnotationRegistry')) {
-                AnnotationRegistry::registerLoader('class_exists');
+                if (method_exists(AnnotationRegistry::class, 'registerUniqueLoader')) {
```

Comment:
The bridge is still PHP 5.3 compatible so this should be replaced by a regular string instead of:: class

Reply:
But: does this actually do something? The first class_exists check just above already ensures unicity, isn't it?

Reply:
The class now has two methods. In v1.6+ `registerUniqueLoader` exists and should be used, but it requires PHP 7.1+.